decidim-proposals/spec/system/valuator_manages_proposals_spec.rb

```diff
@@ -20,7 +20,8 @@
   include_context "when managing a component as an admin"
 
   before do
-    user.update_attribute(:admin, false)
+    user.admin = false
+    admin.save
```

Comment:
🤔 it shouldn't be `user.save`?

Reply:
What the hell, you're right 🤔 how come it's not failing?